Framework/Algorithms/test/ReflectometryReductionOne2Test.h

```diff
@@ -459,14 +459,14 @@ class ReflectometryReductionOne2Test : public CxxTest::TestSuite {
     alg.setProperty("SummationType", "SumInQ");
     alg.setProperty("ReductionType", "DivergentBeam");
     alg.setProperty("ThetaIn", 25.0);
-    MatrixWorkspace_sptr outLam = runAlgorithmLam(alg, 12);
+    MatrixWorkspace_sptr outLam = runAlgorithmLam(alg, 8);
 
-    TS_ASSERT_DELTA(outLam->x(0)[0], 0.934991, 1e-6);
-    TS_ASSERT_DELTA(outLam->x(0)[3], 5.173599, 1e-6);
-    TS_ASSERT_DELTA(outLam->x(0)[7], 10.825076, 1e-6);
-    TS_ASSERT_DELTA(outLam->y(0)[0], 2.768185, 1e-6);
-    TS_ASSERT_DELTA(outLam->y(0)[3], 2.792649, 1e-6);
-    TS_ASSERT_DELTA(outLam->y(0)[7], 2.787410, 1e-6);
+    TS_ASSERT_DELTA(outLam->x(0)[0], 2.347861, 1e-6);
+    TS_ASSERT_DELTA(outLam->x(0)[3], 6.586469, 1e-6);
+    TS_ASSERT_DELTA(outLam->x(0)[7], 12.237946, 1e-6);
+    TS_ASSERT_DELTA(outLam->y(0)[0], 2.776617, 1e-6);
```

Comment:
Same as said previously `y` vs `counts`. I see that the existing test used `y` but work with proper measures if you can in the codebase.